### DISCUSSION TITLE
Zen Browser Twilight 1.0.1-t-20 Update: Bug and Enhancement Suggestions

### DISCUSSION THREAD

**Discussion by Evren-os** [zen-browser/desktop#3170] [Ideas]
Sharing some feedback about the latest Twilight release.

## Environment
- Version: 1.0.1-t-20
- OS: CachyOS

### UI Elements
1. **Vertical Tab Bar URL Spacing**
   * URL bar needs a bit more padding on the left side
   * Currently feels too close to the edge

![1](https://github.com/user-attachments/assets/23ad6e08-5973-4f10-988e-9cebd3a1b4de)

2. **Settings Icon Alignment**
   * Settings gear icon is slightly off
   * Needs minor adjustment towards the right

![2](https://github.com/user-attachments/assets/ce42da36-f0db-418a-bcc8-7d013fe9c041)

3.**Extensions are pinned !!!
   * When the URL bar’s in the vertical tab bar, entering customize toolbar mode shows pinned extensions (after firefox sync). you can’t unpin them, though, they only show up in customize toolbar mode

![241123_22h38m36s_screenshot](https://github.com/user-attachments/assets/c6bbaef6-4172-4a2c-b6a8-958a6f3a9b87)

### Compact Mode Behavior
4. **Toolbar Display Issues**
   * Current behavior:
     - Enabling compact mode (Hide Toolbar) hides the URL bar
     - Disabling it leaves an empty toolbar
     - Browser restart needed to fix the toolbar
     - Not the best user experience

![3](https://github.com/user-attachments/assets/0537f148-22ea-430e-a0c4-b3489bd2242d)

![4](https://github.com/user-attachments/assets/6ba82fd5-6c7c-42de-bf4d-8061cd63d3b8)

### URL Bar Improvements
1. **Button Organization**
   * Consider moving URL bar buttons to the URL popup menu
   * Alternative: Keep only the bookmarks button visible
   * Would create a cleaner interface overall

### Visual Refinements
2. **Top Site Layout**
   * When the toolbar’s off and the URL bar is in the vertical tab bar :
     - Add symmetric padding (top, right, bottom)
     - Would improve overall visual balance

---
Thanks for reading!

**Comment by vraravam** [comment]
Can we also have an option for the sidebar show only icons, but expand on hover? (I thought this was the previous behavior? unable to find it in the latest twilight build)
